src/yarpmotorgui/partitem.h

```diff
@@ -124,7 +124,7 @@ class PartItem : public QWidget
     bool    m_part_motorPositionVisible;
     bool    m_part_dutyVisible;
     bool    m_part_currentVisible;
-    yarp::dev::InteractionModeEnum* m_interactionModes;
```

Comment:
The change is technically correct, but if you look at the whole file, you can see that there are several pointer-vectors and no `std::vector`. This is a change that, in my opinion, makes the code less readable.
I'd rather change all of them or just add the missing delete...

Reply:
Done in https://github.com/robotology/yarp/pull/2774/commits/601570e754d41796bc02931121d08268609a3934